services/src/bin/main.rs

```diff
@@ -58,7 +96,7 @@ fn initialize_logging() -> Option<FileLogWriterHandle> {
         // Thus, we use UTC time instead.
         flexi_logger::DeferredNow::force_utc();
 
-        let (file_writer, fw_handle) = FileLogWriter::builder(file_spec)
+        let (file_writer, _fw_handle) = FileLogWriter::builder(file_spec)
```

Comment:
what changed around here?

Reply:
nothing. I just ignore the handle at a different location in the code. We didn't use it before too